NEWS.txt

```diff
@@ -69,6 +69,7 @@ Interface Changes:
  - optimizer=fast_compile moves computation to the GPU
  - Raise the user stack trace more frequently.
  - Change dev version numbering to follow the PEP 440
+ - round() default to the same as NumPy: half_to_even.
```

Comment:
These are the news for version 0.8. Can you remove them? @notoraptor added them in #5444 as well, so it would avoid a conflict.